### ISSUE TITLE
Bounding Box as input

### ISSUE THREAD

**Issue by chris-hndz** [gaomingqi/Track-Anything#58]
Hello!

Thank you very much again for your good work with this project!

Have you considered incorporating a bounding box as input over the first frame, so that this bounding box selects not only the elements inside it, but also the segmentable elements contained in the bounding box as a set even if they even go a bit out of it, just like the SAM web demo does when selecting a head and incorporates part of the neck outside the bounding box as well?

<img width="54" alt="DemoSAM_face" src="https://user-images.githubusercontent.com/6933154/236677668-048cae26-fdee-4f73-8186-425d97199bed.png">

Source code of the SAM web demo: https://github.com/MiscellaneousStuff/meta-sam-demo

Thanks! :D

**Comment by gaomingqi** [comment]
Hello, thanks for your suggestion! Currently, we have no plan to incorporate box as input since point input can fulfil the requirements above for most cases. We probably consider this shortly and are happy to merge pull requests that further improve TAM's interactivity via the box input. Thanks!

**Comment by licksylick** [comment]
> bounding box as input over the first frame. 

Hello! You can try https://github.com/licksylick/AutoTrackAnything

It creates masks automatically and uses bboxes coordinates as inputs
So basic using: you can detect objects and track them automatically. Hope this helps you :)

**Comment by chris-hndz** [comment]
Hi @licksylick !

Thanks for sharing your project! Can your project track the segmentation of body parts, for example, only the head of a person (face + hair + ears)?

**Comment by licksylick** [comment]
> Thanks for sharing your project! Can your project track the segmentation of body parts, for example, only the head of a person (face + hair + ears)?
  
Of course. All that you need is change model (detector) for this parts (see Readme)

**Comment by chris-hndz** [comment]
Thank you @licksylick for your reply.
We will test your project to see how it behaves with object part tracking.
